html/manifest/serviceworker.json

```diff
@@ -0,0 +1,143 @@
+{
+  "html": {
+    "manifest": {
+      "serviceworker": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/serviceworker",
```

Comment:
I need spec urls to review this. Add them?

Reply:
This manifest member and its sub-features are currently set to `"standard_track": false`, because it exists in a weird situation where it was in the spec, then removed, and now they are waiting for agreement on it being added back to the spec again.